tamr_unify_client/dataset/resource.py

```diff
@@ -84,6 +85,29 @@ def _update_records(self, updates, **json_args):
             .json()
         )
 
+    def upsert_from_dataframe(
+        self, df: pandas.DataFrame, *, primary_key_name: str, ignore_nan: bool = True
+    ) -> dict:
+        """Upserts a record for each row of `df` with attributes for each column in `df`.
+
+        Args:
+            df (pandas.DataFrame): The data to upsert records from.
```

Comment:
`Args` and `Returns` section should not duplicate type information when using type annotations